content/partials/examples/dimmer.slim

```diff
@@ -1,63 +1,64 @@
-.dimmer.demo
-  .ui.segment
-    h3.ui.header
-      |  Dimmable Section
-    .ui.small.images
-      img.ui.wireframe.image[src="/static/kitchen-sink/images/wireframe/image.png"]
-      img.ui.wireframe.image[src="/static/kitchen-sink/images/wireframe/image.png"]
-      img.ui.wireframe.image[src="/static/kitchen-sink/images/wireframe/image.png"]
-    img.ui.medium.wireframe.image[src="/static/kitchen-sink/images/wireframe/media-paragraph.png"]
-    .ui.dimmer
-  .ui.ignored.ignored.icon.buttons
-    .ui.show.button
+.ui
```

Comment:
Also related to the elements filter in the javascript. Without ".ui", the filter will work inconsitently.
http://semantic-ui.com/kitchen-sink.html#/elements
http://semantic-ui.com/kitchen-sink.html#/collections
If you click the code icon in Button and Breadcrumb example on those two page, you can see  that in the Button example, the header "Button" is inside the example segment, while in the BreadCrumb example, the header "Breadcrumb" is outside the example segment. 
This is the main reason to do such modifications.